### ISSUE TITLE
shape_base.py:873: VisibleDeprecationWarning

### ISSUE THREAD

**Issue by pdeperio** [XENON1T/pax#462]
(Benign) warning appearing during MC processing:
```
/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/envs/pax_v6.1.0/lib/python3.4/site-packages/numpy/lib/shape_base.py:873: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
```
Does not appear in Midway processing (```numpy=1.11.1```), so might be from newer version on CVMFS (```numpy=1.11.2```). 

Would be nice to suppress/fix this to clean up logs and facilitate (real) error searching.

Note, to load CVMFS environment on Midway:
```
export PATH="/cvmfs/xenon.opensciencegrid.org/releases/anaconda/2.4/bin:$PATH"
source activate pax_v6.1.0
```

**Comment by JelleAalbers** [comment]
I agree. We'll have to do something like http://stackoverflow.com/questions/22373927/get-traceback-of-warnings to show where the warning actually comes from.

**Comment by tunnell** [comment]
Can we not change version on CVMFS to match Midway for now?  @JelleAalbers are you just suggesting we add `np.seterr(invalid='raise')` in core?

**Comment by JelleAalbers** [comment]
Yes, just to figure out where it comes from. Maybe in plugin.py so it gets loaded everywhere.
